src/pages/Login/Login.tsx

```diff
@@ -0,0 +1,87 @@
+import React, { useState } from 'react';
+import { Box, Button, Paper, TextField, Typography } from '@mui/material';
+import { useLoginMutation } from 'app/api/apiSlice';
+import { useAppDispatch } from 'app/store';
+import { setUser } from 'app/User/userSlice';
+
+const Login = () => {
+    const dispatch = useAppDispatch();
+    const [login, { isLoading }] = useLoginMutation();
+    const [email, setEmail] = useState('');
+    const [password, setPassword] = useState('');
+
+    const handleLogin = async () => {
+        const response = await login({
+            email,
+            password,
+        }).unwrap();
+        dispatch(setUser(response.user));
+    };
+
+    return (
+        <Box
+            sx={{
+                display: 'flex',
+                flex: 1,
+                justifyContent: 'center',
+                alignItems: 'center',
+                background: (theme) => theme.palette.background.secondary,
+            }}
+        >
+            <Paper
+                elevation={0}
+                sx={{
+                    display: 'flex',
+                    flexDirection: 'column',
+                    justifyContent: 'center',
+                    alignItems: 'center',
+                    paddingY: (theme) => theme.spacing(4),
+                    paddingX: (theme) => theme.spacing(5),
+                    borderRadius: '5px',
+                    border: (theme) =>
+                        `1px solid ${theme.palette.border.neutral}`,
+                }}
+            >
+                <Typography
+                    variant="regular"
+                    sx={{ marginBottom: (theme) => theme.spacing(2) }}
+                >
+                    Welcome to Music Manager!
```

Comment:
We could use https://react.i18next.com/ for texts in the app. Or just create a file with all the messages - it will be easier to control them

Reply:
I implemented translations management using the library you mentioned. Later I will add a button in the app to change the language